### ISSUE TITLE
NPM install build system failing

### ISSUE THREAD

**Issue by GitMiester** [twbs/bootstrap#19601]
Hello,

All I am trying to do is use Flexbox with v4.  I can't find a flexbox download anywhere.  Is there one?  http://v4-alpha.getbootstrap.com/getting-started/download/ doesn't seem to work even when I click 
the Flexbox option at the bottom of the page.  It downloads but doesn't seem to contain flexbox.

So after looking around I decided to run the build system and download bootstrap via npm install. 

npm --version
3.8.2

nodejs --version                                                                                                                                                     
v0.10.37

Theres a lot of missing files like grunt/sauce_browsers.yml and the whole folders files /grunt/bs-sass-compile/* . 

Also when I run 

```bundle install```  

I get the following output:

```
Could not locate Gemfile or .bundle/ directory
```

Can someone please link to boostrap 4 flexbox?  The npm install is hard and is having issues.

**Comment by cvrebert** [comment]
> Theres a lot of missing files like grunt/sauce_browsers.yml and the whole folders files /grunt/bs-sass-compile/* .

Duplicate of #16400.

> Could not locate Gemfile or .bundle/ directory

The Gemfile is only used to install Jekyll and SCSS-Lint, which aren't necessary in normal npm usage.

**Comment by cvrebert** [comment]
> I can't find a flexbox download anywhere. Is there one?

Not yet. See #18292.

**Comment by GitMiester** [comment]
Thanks for the reply.  I saw that it had been closed and effects v3 so I opened this issue up.

I went ahead and downloaded v4 here on github instead of downloading it via npm install.  Grunt worked flawlessly.  

Thanks for the heads up about Jekyll.  

Also if anyone gets stuck in this situation heres a flexbox mirror in the mean time https://github.com/GitMiester/bootstrap-4-flexbox .  I wasn't sure if the JS needed to be added for flexbox but put it there anyways.
